ci/test/06_script_b.sh

```diff
@@ -36,7 +36,7 @@ fi
 
 if [ "${RUN_TIDY}" = "true" ]; then
   export P_CI_DIR="${BASE_BUILD_DIR}/bitcoin-$HOST/src/"
-  CI_EXEC run-clang-tidy "${MAKEJOBS}"
+  CI_EXEC run-clang-tidy -quiet "${MAKEJOBS}"
```

Comment:
nit what about discarding anything but errors?

```suggestion
  ( CI_EXEC run-clang-tidy -quiet "${MAKEJOBS}" ) | grep "error or so"
```

Reply:
This sounds great.

Reply:
Taken this. Passing `-C5` should provide enough additional context as well.